modules/mono/editor/GodotTools/GodotTools/Build/DotNetFinder.cs

```diff
@@ -61,7 +62,7 @@ public static string FindDotNetExe()
             process.StartInfo.EnvironmentVariables["DOTNET_CLI_UI_LANGUAGE"] = "en-US";
 
             var lines = new List<string>();
-
+            process.StartInfo.StandardOutputEncoding = Encoding.UTF8;
```

Comment:
Same as for the other file, define this directly in the `ProcessStartInfo` constructor instead of setting it up after the fact and between the definition and use of `lines`.
```
using Process process = new Process();
            process.StartInfo = new ProcessStartInfo(dotNetExe, "--list-sdks")
            {
                UseShellExecute = false,
                RedirectStandardOutput = true,
                StandardOutputEncoding = Encoding.UTF8
            };
```